jdbc/src/main/java/org/apache/zeppelin/jdbc/JDBCInterpreter.java

```diff
@@ -878,6 +900,17 @@ public int getMaxResult() {
     return maxLineResults;
   }
 
+  public int getMaxRows() {
+	return maxRows;
+  }
+
+  /**
+   * @return the rows fetch size
+   */
+  public int getRowsFetchSize() {
+	return rowsFetchSize;
+  }
+
```

Comment:
Since these 2 fields are not used out of `JDBCInterpreter.java`, we can remove these 2 get methods